src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareUtils.cs

```diff
@@ -28,6 +28,9 @@ internal static DacDeployOptions CreateSchemaCompareOptions(DeploymentOptions de
             PropertyInfo[] deploymentOptionsProperties = deploymentOptions.GetType().GetProperties();
```

Comment:
(I couldn't add this comment on an earlier line, but this is meant for line 26) Maybe we should consider renaming this function from `CreateSchemaCompareOptions()` to `CreateDacDeployOptions()` since this isn't used in only schema compare anymore. Doesn't have to be changed in this PR, but just wanted to bring it up since I noticed it :)

Reply:
I Agree with you! I'll create a task for this one and make sure to update all required calls changes to generic method names, Thanks :)

Reply:
https://github.com/microsoft/sqltoolsservice/issues/1554 created on STS github issues board